OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/UserState.kt

```diff
@@ -0,0 +1,39 @@
+package com.onesignal.user
+
+import com.onesignal.common.IDManager
+import org.json.JSONObject
+
+/**
+ * A user state.
+ */
+class UserState {
```

Comment:
Consider making non-optional and default to `""` to match push subscription state and simplify `toJSONObject()` method.